.github/workflows/kvrocks.yaml

```diff
@@ -65,10 +65,10 @@ jobs:
       matrix:
         include:
           - name: Darwin Clang
-            os: macos-latest
```

Comment:
@PragmaTwice Why do we switch macOS to 11

Reply:
macos-11 is equal to macos-latest, referring to https://github.com/actions/runner-images#available-images

I used to try macos-12, but I reverted the change later.

Reply:
Okay, thanks